store-implementation/hbase-store/src/main/java/uk/gov/gchq/gaffer/hbasestore/HBaseStore.java

```diff
@@ -180,6 +184,25 @@ protected SchemaOptimiser createSchemaOptimiser() {
     }
 
     @Override
+    public void validateSchemas() {
+        super.validateSchemas();
+        if (!getSchema().getVertexSerialiser().isConsistent()) {
+            throw new SchemaException("Vertex serialiser is inconsistent - store may be unable to handle this.");
+        }
+    }
+
+    @Override
+    protected void validateSchemaElementDefinition(final Entry<String, SchemaElementDefinition> schemaElementDefinitionEntry, final ValidationResult validationResult) {
+        super.validateSchemaElementDefinition(schemaElementDefinitionEntry, validationResult);
+
+        for (String property : schemaElementDefinitionEntry.getValue().getGroupBy()) {
+            if (!schemaElementDefinitionEntry.getValue().getPropertyTypeDef(property).getSerialiser().isConsistent()) {
+                validationResult.addError("Property serialiser is inconsistent - store may be unable to handle this.");
```

Comment:
Please could you update the error message to:

"Serialiser for groupBy property " + property + " is inconsistent. This store requires all groupBy property serialisers to be consistent."